src/commands/projects/remove.js

```diff
@@ -0,0 +1,93 @@
+const { TwilioClientCommand } = require('@twilio/cli-core').baseCommands;
+const { TwilioCliError } = require('@twilio/cli-core').services.error;
+
+class ProjectsRemove extends TwilioClientCommand {
+  async runCommand() {
+    this.removeProjectStatus(this.userConfig.getProjectById(this.args.project), this.args.project);
```

Comment:
Should just store this locally and pass it around: `const deleteProject = this.userConfig.getProjectById(this.args.project);`